### DISCUSSION TITLE
Need GPU cloud add dreamshaperXL-v21 and Realistic Vision V6.0 B1 checkpoints

### DISCUSSION THREAD

**Discussion by syl1130** [Acly/krita-ai-diffusion#986] [General]
Hello Acly @Acly , 

Because I purchased an online service, but I want to use dreamshaperXL-v21TurboDPMSDE.safetensors for the checkpoint XL large model. [https://civitai.com/models/112902/dreamshaper-xl](url) and the SD1.5 model of Realistic Vision V6.0 B1 is also very good.[https://civitai.com/models/4201/realistic-vision-v60-b1](url)

Can GPU cloud servers be added? Convenient for users to purchase cloud servers and switch to Dreamshaper XL large models. Thanks

Best Regards,
Alxen

**Comment by syl1130** [comment]
Many users hope to use the latest checkpoint model on GPU Cloud Service

**Comment by Acly** [comment]
I agree some checkpoints need to be updated.

Adding bigger selection of checkpoints is tricky, as it negatively affects queue/wait time when there are usage spikes. Probably will look into adding way to use custom Lora first.

**Reply by syl1130** [reply]
@Acly  
Thank you very much for agreeing to update and add checkpoints. So, let's add this dreamshaperXL-v21TurboDPMSDE.safetensors. I have been using this checkpoint mostly in my work. thanks

It would be great if users could add and use custom Lora files.
